File: RxAndroidRetrofit/app/src/main/java/com/example/rxandroidretrofit/APIUtils.java
package com.example.rxandroidretrofit;

public class APIUtils {
    public static final String Base_Url = "https://api.learn2crack.com/";

    public  static DataClient getData(){
        return RetrofitCilent.getClient(Base_Url).create(DataClient.class);
    }
}

File: LoiCuaBac2/app/src/main/java/com/example/loicuabac2/service/retrofit/APIUtils.kt
package com.example.loicuabac2.service.retrofit

import com.example.loicuabac2.Constants

class APIUtils {


    object data{
        fun getData(): DataClient {
            return RetrofitClient.RetrofitgetClient.getClient(Constants.URL_SERVER).create(DataClient::class.java)
        }
    }

}